src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -97,13 +97,12 @@ private void MakeCompilationOutputRunnableForCoreCLR()
         {   
             WriteDepsFileAndCopyProjectDependencies(_exporter);
 
-            var emitEntryPoint = _compilerOptions.EmitEntryPoint ?? false;
+            var isRunnable = _compilerOptions.EmitEntryPoint ?? _context.ProjectFile.OverrideIsRunnable;
 
-            if (emitEntryPoint && !_context.IsPortable)
+            if (isRunnable && !_context.IsPortable)
             {
-                // TODO: Pick a host based on the RID
                 VerifyCoreClrPresenceInPackageGraph();
-                CoreHost.CopyTo(_runtimeOutputPath, _compilerOptions.OutputName + Constants.ExeSuffix);
```

Comment:
I think this was changed during an earlier prototype when all the dependencies were being copied to the build output folder.  I'm reverting this change.

---
In reply to: [71799445](https://github.com/dotnet/cli/pull/3911#discussion_r71799445) [](ancestors = 71799445,71798032,71797819,71797731,71797574)